radio/src/tasks.cpp

```diff
@@ -52,6 +52,11 @@ TASK_FUNCTION(menusTask)
 
   mixerTaskInit();
 
+#if !defined(COLORLCD)
+  // Ensure correct init on B&W if starting on 'Channels Monitor' page.
+  pushEvent(EVT_ENTRY);
```

Comment:
We should probably really start splitting the UI handling code cleanly so it does not spread to such places.